### PR TITLE
Reduce tile view pages chunk size, fix minor display issues

### LINKED ISSUE #869: Marketplace and Catalog chunk sizes are large and should be reduced.
Chunk sizes for Catalog and Marketplace are larger than other chunks and have special settings in the performance scenario tests. These should be reduced and these exceptions removed or reduced.

@alecmerdler

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Reduces the chunk size be eliminating the use of _.intersection in the tile views (was adding about 80k) and updating to latest patternfly-react-extensions which contains some dependency fixes.

Removed overrides for the tile description fader as this was fixed in patternfly-react-extensions.

Updated the tile view pages to always be full page rather than jumping when filtering.

Bumps patternfly-react-extensions to 2.13.11

Fixes:  https://bugzilla.redhat.com/show_bug.cgi?id=1640054

Fixes #869 

@sosiouxme fyi

### CODE REVIEW

**Comment by spadgett** [comment]
This seems odd to me since `_.intersection` is already in vendors~main best I can tell.

screen shot 2018-11-30 at 8 43 43 am

@alecmerdler fyi

**Comment by jeff-phillips-18** [comment]
Narrowed it down to this line: https://github.com/openshift/console/pull/884/files#diff-ed1b361414098f1230da28a2b0464badR220

Without that change, the chunk size reported by https://github.com/openshift/console/pull/884/files#diff-70660d6a99f79632dbb209f371ff407cR99 is +81k

**Comment by sosiouxme** [comment]
Thanks for the heads up, I'll know where to look if the new automation for downstream sources has any hitches.

File: frontend/integration-tests/tests/performance.scenario.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ describe('Performance test', () => {
       expect(routeChunk).not.toBeNull();
       // FIXME: Really need to address this chunk size
       if (routeName === 'catalog' || routeName === 'marketplace') {
-        expect((routeChunk as any).decodedBodySize).toBeLessThan(200000);
+        expect((routeChunk as any).decodedBodySize).toBeLessThan(100000);
```
Comment (alecmerdler) [inline frontend/integration-tests/tests/performance.scenario.ts]: Awesome!

**Pull request merged**